release/release.md

```diff
@@ -0,0 +1,122 @@
+# How to do a Kata Containers Release
+
+* [Introduction](#Introduction)
+* [Requirements](#Requirements)
+* [Release process](#release-process)
+
+# Introduction
+
+If you are reading this document, you might want to create a Kata Containers
+Release.
+
+The Kata Containers Release Process is defined in the following
+[document][release-process-definition]. To simplify this process, we have
+created a [Release Checklist][release-checklist]. Finally, to simplify the
+Release Checklist process we have automated most of the process, this document
+guides on how to use release scripts instead of do all the checklist manually.
+
+
+## Requirements
+
+- It is recommended to have at least 2 GB of free disk space to perform these tasks.
+
+- [Go](https://golang.org/doc/install#)
+
+- [hub](https://github.com/github/hub)
+
+- OBS account with permissions on /home:katacontainers (https://build.opensuse.org/project/subprojects/home:katacontainers)
+
+- GitHub permissions to push tags and creates Releases in Kata repositories.
+
+- GPG configured to sign git tags. https://help.github.com/articles/generating-a-new-gpg-key/
+
+- You should configure your GitHub to use your ssh keys (to push to branches). See https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/.
+    * As an alternative, configure hub to push and fork with https, `git config --global hub.protocol https` (Not tested yet) *
+
+- [Docker](https://docs.docker.com/install/)
+
+- Get the [Packaging](https://github.com/kata-containers/packaging) Kata repository
+
+  ```bash
+  $ go get -d github.com/kata-containers/packaging
+  ```
+
+## Release process
+
+Notes:
+
+- The steps described here are safe to repeat more than one time, it is safe to
+  repeat in case of unexpected issues. And is it not required start from the
+  beginning.
+
+
+- It is "safe" to run this process on any machine. It creates all assets in
+  sub-directories and should not modify the entire system.
+
+```bash
+$ cd ${GOPATH}/src/github.com/kata-containers/packaging
+# make sure you are up-to-date.
+$ git pull
+```
+1. Bump repositories
+  ```bash
```

Comment:
Nit: Could you indent all the code blocks by 1 space as currently they are "out-dented" whereas they should be indented below the numbers. I think you need to do something like:

```sh

1. Bump repositories
    ```bash
    $ cd ${GOPATH}/src/github.com/kata-containers/packaging/release
       :

    ```

```